packages/vue-cli-plugin-typescript/generator/template/src/utils/envUtils.ts

```diff
@@ -3,7 +3,7 @@
  *
  * @returns true if the .env file is valid. false otherwise.
  */
-export function isEnvValid() {
+export function isEnvValid(): boolean {
```

Comment:
Check the https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards

It should allow the typescript compiler to understand that "after isEnv is used, process.env has those keys"